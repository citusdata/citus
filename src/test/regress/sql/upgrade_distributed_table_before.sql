CREATE SCHEMA upgrade_distributed_table_before;
SET search_path TO upgrade_distributed_table_before, public;

CREATE TABLE t(a int);
SELECT create_distributed_table('t', 'a');
INSERT INTO t SELECT * FROM generate_series(1, 5);
