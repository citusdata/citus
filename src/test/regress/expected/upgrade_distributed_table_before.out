CREATE SCHEMA upgrade_distributed_table_before;
SET search_path TO upgrade_distributed_table_before, public;
CREATE TABLE t(a int);
SELECT create_distributed_table('t', 'a');
 create_distributed_table 
--------------------------
 
(1 row)

INSERT INTO t SELECT * FROM generate_series(1, 5);
